shoop/testing/factories.py

```diff
@@ -206,9 +206,11 @@ def post(self, create, extracted, **kwargs):
             sp.categories = shop.categories.all()
 
 
-def get_address(**overrides):
+def get_address(immutable=False, **overrides):
     data = dict(DEFAULT_ADDRESS_DATA, **overrides)
-    return Address(**data)
+    if immutable:
+        return ImmutableAddress.objects.create(**data)
+    return MutableAddress.objects.create(**data)
```

Comment:
no wait.. hmm.. that would return abstract `Address` object, if not immutable and `MutableAddress.from_data(data, immutable=True)` does not seem right... we should rethink the API a bit...